Library/Homebrew/cmd/formulae.sh

```diff
@@ -0,0 +1,11 @@
+#:  * `formulae`
+#:
+#:  Prints a sorted list of locally available formulas including shortnames.
+#:
+
+# shellcheck disable=SC2155
+homebrew-formulae() {
+  local formulae="$(find "$HOMEBREW_REPOSITORY"/Library/Taps -type d \( -name Casks -o -name cmd -o -name .github \) -prune -false -o -name '*rb' | sed 's/\.rb//g' | sed -E 's .*/Taps/(.*)/homebrew- \1/ g' | sed 's /Formula/ / g')"
+  local shortnames="$(echo "$formulae" | cut -d / -f 3)"
+  echo -e "$formulae \n $shortnames" | grep -v '^homebrew/' | sort -uf
```

Comment:
Could the `grep-v` instead be part of one of the previous lines?

Reply:
I don't think so.  All the formula are only listed by their full name according to the tap folder structure (so, homebrew/core/<formula names>).  If those lines are removed before the `cut -d / -f 3` command (which removes everything before the final `/`), then none of the homebrew core formula will get included.  

I actually thought about doing it this way for a while, but this is by far the most efficient method I could come up with.    By removing the extended name of the core homebrew formula in the list, then those names can be generated with the short names of everything else and simply have those longer names omitted at the end (`grep -v`).  Otherwise, an entire separate 'pipeline' so to speak is needed just for the special case of the homebrew core names.

Reply:
Ok, makes sense 👍🏻